cmd/ctr/commands/run/run_unix.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func init() {
-	Command.Flags = append(Command.Flags, cli.BoolFlag{
+	ContainerFlags = append(ContainerFlags, cli.BoolFlag{
```

Comment:
That should have work just fine the old way, no?

Reply:
No, it was not exposed for `ctr c create`: https://github.com/containerd/containerd/blob/e6a3dd3550a69d6fe8bb3c5effeeb3a566d7a027/cmd/ctr/commands/containers/containers.go#L52